packages/cli/src/commands/spaces/trusted-ips/remove.ts

```diff
@@ -0,0 +1,46 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import heredoc from 'tsheredoc'
+
+export default class Remove extends Command {
+  static aliases = ['trusted-ips:remove']
+  static topic = 'trusted-ips'
+  static description = 'Remove a range from the list of trusted IP ranges'
+  static help = 'Uses CIDR notation.'
```

Comment:
Add this as a second line to description to show on the `--help` output.

Reply:
Done. Thanks you!